docs/tutorials/index.md

```diff
@@ -0,0 +1,186 @@
+- how to build event-driven applications on continuous streams
+- how Flink is able to provide fault-tolerant, stateful stream processing with exactly-once semantics
+
+These tutorials focus on four critical concepts: continuous processing of streaming data, event
+time, stateful stream processing, and state snapshots. This page introduces these concepts.
+
+{% info Note %} Accompanying these tutorials are a set of hands-on exercises that will guide you
+through learning how to work with the concepts being presented.
+
+{% top %}
+
+## Stream Processing
+
+Streams are data's natural habitat. Whether it's events from web servers, trades from a stock
+exchange, or sensor readings from a machine on a factory floor, data is created as part of a stream.
+But when you analyze data, you can either organize your processing around _bounded_ or _unbounded_
+streams, and which of these paradigms you choose has profound consequences.
+
+<img src="{{ site.baseurl }}/fig/bounded-unbounded.png" alt="Bounded and unbounded streams" class="offset" width="90%" />
+
+**Batch processing** is the paradigm at work when you process a bounded data stream. In this mode of
+operation you can choose to ingest the entire dataset before producing any results, which means that
+it's possible, for example, to sort the data, compute global statistics, or produce a final report
+that summarizes all of the input.
+
+**Stream processing**, on the other hand, involves unbounded data streams. Conceptually, at least,
+the input may never end, and so you are forced to continuously process the data as it arrives. 
+
+In Flink, applications are composed of **streaming dataflows** that may be transformed by
+user-defined **operators**. These dataflows form directed graphs that start with one or more
+**sources**, and end in one or more **sinks**.
+
+<img src="{{ site.baseurl }}/fig/program_dataflow.svg" alt="A DataStream program, and its dataflow." class="offset" width="80%" />
+
+Often there is a one-to-one correspondence between the transformations in the programs and the
+operators in the dataflow. Sometimes, however, one transformation may consist of multiple operators.
+
+An application may consume real-time data from streaming sources such as message queues or
+distributed logs, such as Apache Kafka or Kinesis. But flink can also consume bounded, historic data
+from a variety of data sources. Similarly, the streams of results being produced by a Flink
+application can be sent to a wide variety of systems, and the state held within Flink can be
+accessed via a REST API.
+
+<img src="{{ site.baseurl }}/fig/flink-application-sources-sinks.png" alt="Flink application with sources and sinks" class="offset" width="90%" />
+
+### Parallel Dataflows
+
+Programs in Flink are inherently parallel and distributed. During execution, a
+*stream* has one or more **stream partitions**, and each *operator* has one or
+more **operator subtasks**. The operator subtasks are independent of one
```

Comment:
the new term for "subtask" is "task", but, according to the [glossary](https://ci.apache.org/projects/flink/flink-docs-master/concepts/glossary.html#sub-task), a "Sub-Task" is the same but "emphasizes that there are multiple parallel Tasks for the same Operator or Operator Chain". I think, we should use "Task" here (not sure about capitalization).

Reply:
But here we are, in fact, emphasizing the parallel nature of the many sub-tasks for each operator. If we don't use the term here, then where would we use it?

Reply:
Yes, you are right - should it be consistently "sub-task" as in the glossary? (I would also be fine with "subtask" here since it improves the flow)